src/Forms/FormConfig.php

```diff
@@ -18,6 +18,8 @@ class FormConfig implements FieldContainerInterface
     const OVERRIDE_SUBMIT      = 'submitButton';
     const OVERRIDE_PANEL_CLASS = 'panelClass';
 
+    const DEFAULT_NAVIGATION_HEADING_LEVEL = 'h3';
```

Comment:
Leave the default div, override in local version of FormConfig

Reply:
Got it!